.prettierrc

```diff
@@ -1,4 +1,5 @@
 {
+  "trailingComma": "es5",
```

Comment:
Was this so that the formatted source would have trailing commas? If so we should probably pass it explicitly [here](https://github.com/utlime/webpack-retry-chunk-load-plugin/blob/master/src/index.js#L73) when using the prettier api. I think it's ok for the actual source of the plugin to have trailing commas as it's been supported since node 8.

Reply:
Hi, sorry for late answer.
Added for babel and resolved conflicts.